migrace/03_migrace_1.sql

```diff
@@ -13,7 +13,6 @@ CREATE SEQUENCE public.historie_akce_seq
     NO MINVALUE
     NO MAXVALUE
     CACHE 1;
-ALTER TABLE public.historie_akce_seq OWNER TO cz_archeologickamapa_api;
```

Comment:
Tady mi není jasný důvod.

Reply:
Nakonec je jednodušší vlastníka dropnout při záloze produkce a použít existujícího uživatele na cílovém serveru, jinak tam vznikaly různé potíže a kroky navíc, nebo bych to musel dělat celé pod cz_archeologickamapa_api a zbytečně ho zakládat, i když ho nepotřebujeme... Tady to třeba řvalo, že sekvence musí mít stejného vlastníka jako tabulka.